config/main.py

```diff
@@ -313,6 +313,14 @@ def _abort_if_false(ctx, param, value):
     if not value:
         ctx.abort()
 
+def _get_optional_services():
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    optional_services_dict = config_db.get_table('FEATURE')
+    if not optional_services_dict:
+        return None
```

Comment:
@shine4chen please fix a bug:
```bash
TASK [execute cli "config load_minigraph -y" to apply new minigraph] ***********
Tuesday 25 February 2020  19:54:51 +0000 (0:00:00.067)       0:00:18.526 ****** 

fatal: [vlab-01]: FAILED! => {"changed": true, "cmd": "config load_minigraph -y", "delta": "0:00:55.103499", "end": "2020-02-25 19:55:46.147827", "msg": "non-zero return code", "rc": 1, "start": "2020-02-25 19:54:51.044328", "stderr": "Traceback (most recent call last):\n  File \"/usr/bin/config\", line 12, in <module>\n    sys.exit(config())\n  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 722, in __call__\n    return self.main(*args, **kwargs)\n  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 697, in main\n    rv = self.invoke(ctx)\n  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 1066, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 895, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 535, in invoke\n    return callback(*args, **kwargs)\n  File \"/usr/lib/python2.7/dist-packages/config/main.py\", line 684, in load_minigraph\n    _stop_services()\n  File \"/usr/lib/python2.7/dist-packages/config/main.py\", line 451, in _stop_services\n    for service in _get_optional_services():\nTypeError: 'NoneType' object is not iterable", "stderr_lines": ["Traceback (most recent call last):", "  File \"/usr/bin/config\", line 12, in <module>", "    sys.exit(config())", "  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 722, in __call__", "    return self.main(*args, **kwargs)", "  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 697, in main", "    rv = self.invoke(ctx)", "  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 1066, in invoke", "    return _process_result(sub_ctx.command.invoke(sub_ctx))", "  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 895, in invoke", "    return ctx.invoke(self.callback, **ctx.params)", "  File \"/usr/lib/python2.7/dist-packages/click/core.py\", line 535, in invoke", "    return callback(*args, **kwargs)", "  File \"/usr/lib/python2.7/dist-packages/config/main.py\", line 684, in load_minigraph", "    _stop_services()", "  File \"/usr/lib/python2.7/dist-packages/config/main.py\", line 451, in _stop_services", "    for service in _get_optional_services():", "TypeError: 'NoneType' object is not iterable"], "stdout": "Stopping service swss ...\nStopping service lldp ...\nStopping service pmon ...\nStopping service bgp ...\nStopping service hostcfgd ...\nStopping service nat ...", "stdout_lines": ["Stopping service swss ...", "Stopping service lldp ...", "Stopping service pmon ...", "Stopping service bgp ...", "Stopping service hostcfgd ...", "Stopping service nat ..."]}
```
@qiluo-msft / @yxieca FYI.

Reply:
It seems github website has some error that we can't open PRs now. I will fix it after github service recover.